cmd/testclient/client.go

```diff
@@ -650,13 +652,13 @@ func main() {
 		})
 		if err != nil {
 			sugar.Errorw("error copying snapshot", "error", err)
-			os.Exit(1)
+			return
 		}
 		sugar.Infow("snapshot copied", "response", v)
 
 	default:
-		sugar.Infow("error: invalid method", "method", *method)
-		os.Exit(1)
+		sugar.Errorw("error: invalid method", "method", *method)
```

Comment:
Can you please update sugar to netbookai logger??

Reply:
no use, would be just a code noise with the context and all,

Reply:
yeah but it will consistent eveywhere in spawner that is reason we are using logging interface